src/components/radios/index.md.njk

```diff
@@ -35,6 +35,18 @@ If there are only 2 options, you can either stack the radios or display them inl
 
 {{ example({group: "components", item: "radios", example: "default", html: true, nunjucks: true, open: true, size: "s"}) }}
 
+### Radio items with hints
+
+You can add a hint to a radio item, explaining what the option means.
+
+{{ example({group: "components", item: "radios", example: "hint", html: true, nunjucks: true, open: true, size: "s"}) }}
+
+### Radio items with a text divider
+
+You can add a text divider to separate available options
```

Comment:
In the Service Manual it has the following copy:

"If some radio button or checkbox labels are phrased differently from the others, it may make the page more readable if you separate the options that are different with an ‘or’."

https://www.gov.uk/service-manual/design/designing-good-questions

Reply:
ok will use that copy then

Reply:
used a tweaked version of that sentence